message_sender/serializers.py

```diff
@@ -1,34 +1,20 @@
 from .models import Inbound, Outbound, OutboundSendFailure, Channel
 from rest_hooks.models import Hook
 from rest_framework import serializers
-from rest_framework.exceptions import APIException
-
-
-class IncorrectChannel(APIException):
-    status_code = 400
-    default_detail = 'The channel supplied is not configured.'
-    default_code = 'incorrect_channels'
 
 
 class OutboundSerializer(serializers.HyperlinkedModelSerializer):
 
+    channel = serializers.PrimaryKeyRelatedField(
+        queryset=Channel.objects.all(), required=False)
```

Comment:
I'm not sure if this will cause issues. This will get evaluated when the program is first run, but not again. So if we start the program, then add a channel, that new channel will not be reflected in this list. Is it possible to put `Channel.objects.all` here instead? If not, we might have to rethink how we do this.